docs/production/mobile-push-notifications.md

```diff
@@ -44,51 +50,19 @@ You can enable this for your Zulip server as follows:
    docker exec -it -u zulip <container_name> /home/zulip/deployments/current/manage.py register_server
    ```
 
-   This command will print the registration data it would send to the
-   mobile push notifications service, ask you to accept the terms of
-   service, and if you accept, register your server. If you have trouble,
-   email support@zulip.com with the output of this command.
+   This command will print the registration data it would send to the mobile
+   push notifications service, ask you to accept the terms of service, and if
+   you accept, register your server. If you have trouble, [contact Zulip
+   support](https://zulip.com/help/contact-support) with the output of this
+   command.
 
 1. If you or your users have already set up the Zulip mobile app,
    you'll each need to log out and log back in again in order to start
    getting push notifications.
```

Comment:
"Log out and back in, in the mobile app."  We should clarify, because logging out and back in, in the webapp, won't help at all.

Reply:
Done.